src/components/common/YourCompany/AssociationType.vue

```diff
@@ -233,29 +237,30 @@ export default class AssociationType extends Mixins(CommonMixin, EnumMixin) {
   }
 }
 
-/** Below brought over from business-create-ui.
-    remove extra space taken by error message */
-::v-deep .v-text-field__details {
-  margin-bottom: -8px !important;
+// allow selection to be full height so tail of "g" is visible
+:deep(.v-select__selections) {
+  line-height: unset;
 }
 
-// Vuetify Overrides
-::v-deep .v-list-item .v-list-item__title, .v-list-item .v-list-item__subtitle {
-  color: $gray7;
+// remove extra space taken by error message
+:deep(.v-text-field__details) {
+  margin-bottom: -8px !important;
 }
 
-::v-deep .v-list-item--link:hover:not(.v-list-item--active) {
-  background-color: $gray1;
-  color: $app-blue !important;
-}
+// Vuetify Overrides
+:deep() {
+  .v-list-item .v-list-item__title {
+    color: $gray7;
+  }
 
-::v-deep .v-list-item:hover {
-  .v-list-item__title {
+  .v-list-item--link:hover:not(.v-list-item--active) {
+    background-color: $gray1;
     color: $app-blue !important;
   }
-}
 
-::v-deep .v-list-item--active .v-list-item__title, .v-list-item .v-list-item__subtitle {
```

Comment:
I removed the `.v-list-item .v-list-item__subtitle` selector as I couldn't find any elements with that class.

I tested this and it looks good to me as is.